### ISSUE TITLE
New concept EXPLODE

### ISSUE THREAD

**Issue by Tarotis** [concepticon/concepticon-data#1245]
There are currently three unmapped entries of "explode", two of those including "to burst". Further, there are various "to burst" mapped to "BREAK (BREAKING)" right now. An additional case of "to burst, explode", will be added in the Kaufman list, making it four unmapped "explodes", one burst [english]; éclater [french], and two "burst, break" currently mapped to "[BREAK (DESTROY OR GET DESTROYED)](https://concepticon.clld.org/parameters/680)". The latter seems quite difficult: Where does it fit in? We could distinguish "burst, explode" from "burst, break", but what if an entry is only "burst", like [this](https://concepticon.clld.org/values/Lundgren-2020-275-47).

**Comment by AnnikaTjuka** [comment]
My approach would be to leave these tricky mappings aside for now and focus on the mappings that are currently available as Concepticon concept sets. The workflow should not be to go through the list, look up every gloss that is not mapped, and find out if other lists have the same gloss. At one point, we also discussed creating an automatic workflow for this. Splitting and adding new concepts also has implications for other data collections, so it is more of an overarching process rather than looking at each list individually.

**Comment by LinguList** [comment]
Yes. It requires so e good idea on intentions of glossers, etc. If we had clear definitions and knew the explode instances cover German explodieren with a clear concept being denoted, mapping would be okay with a new concept explode, if that does not exist yet.

**Comment by Tarotis** [comment]
In the three unmapped cases, I think we would be save to map to `EXPLODE`, which does not yet exist (neither does something similar). In two cases, it co-exists with "break", so they are clearly differentiated.

In cases where it's only "burst" or "burst, break", I'd probably keep mapping to `BREAK (BREAKING)`, which seems less specific than `EXPLODE`.

**Comment by LinguList** [comment]
Okay.
